packages/pfff/pfff.0.37.6/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Tools and APIs for program analysis, code visualization, refactoring"
+description: """
+pfff is a set of tools and APIs to perform some static analysis,
+dynamic analysis, code visualizations, code navigations, or
+style-preserving source-to-source transformations such as refactorings
+on source code. For now the effort is focused on PHP but there is also
+good support for C, C++, Java, HTML, Javascript, Css, and preliminary
+support for Erlang, Python, C#, Lisp, Scheme, Haskell, OPA, Sql, and
+even TeX. There is also very good support for OCaml and noweb
+(literate programming) so that pfff can be used on the code of pfff
+itself.
+
+For more information see https://github.com/facebook/pfff/wiki/Main"""
+
+maintainer: "yoann.padioleau@gmail.com"
+authors: [ "Yoann Padioleau <yoann.padioleau@gmail.com>" ]
+homepage: "https://github.com/facebook/pfff/wiki/Main"
+bug-reports: "https://github.com/aryx/pfff/issues"
+dev-repo: "git+https://github.com/aryx/pfff.git"
+
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
+  [make "install-libs"]
+]
+remove: [
+  [make "uninstall-libs"]
+]
+depends: [
+  "ocaml" {>= "4.03.0" & < "4.09.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.03.0" & < "4.09.0"}
  "ocamlfind" {build}
```
The installation step makes use of `ocamlfind` but it is not listed in the dependencies.